packages/components/src/components/typography/Text/Text.tsx

```diff
@@ -18,7 +21,11 @@ type ColorProps = {
     $color?: string;
 };
 
-const getColor = ({ $variant, theme, $color }: ColorProps): CSSColor | 'inherit' | string => {
+export const getColorForTextVariant = ({
```

Comment:
exporting this is also a bit controversial, @jvaclavik we may need to discuss ho to work with this and what is proper place for this function